index.js

```diff
@@ -321,4 +372,56 @@ input DeleteOtherAnswerInput {
   parentAnswerId: ID!
 }
 
+input CreateRoutingRuleSetInput {
+  questionPageId: ID!
+}
+
+input UpdateRoutingRuleSetInput {
+  id: ID!
+  questionPageId: ID!
+}
+
+input DeleteRoutingRuleSetInput {
+  id: ID!
+}
+
+input CreateRoutingRuleInput {
+  routingRuleSet: ID!
+  operation: RoutingOperation
+  conditions: [ID]
+  goto: ID
+}
+
+input UpdateRoutingRuleInput {
+  id: ID!
+  operation: RoutingOperation
+  conditions: [ID]
+  goto: ID
+}
+
+input DeleteRoutingRuleInput {
+  id: ID!
+}
+
+input CreateRoutingConditionInput {
+  comparator: RoutingComparator!
+  answerId: ID!
+  routingValue: RoutingConditionValueInput!
+}
+
+input UpdateRoutingConditionInput {
+  id: ID!
+  comparator: RoutingComparator
+  answerId: ID
+  routingValue: RoutingConditionValueInput
+}
+
+input DeleteRoutingConditionInput {
+  id: ID!
+}
+
+input RoutingConditionValueInput {
+  intValue: Int
+  intArrayValue: [Int]
+}
```

Comment:
Can you not use union types for inputs?

Reply:
No, unfortunately not. GraphQL unions need to be Object types, not Input types.